tests/integration_tests/plugin_tests/test_pvt_plot.py

```diff
@@ -4,7 +4,6 @@
 
 def test_pvt_plot(dash_duo, app, shared_settings) -> None:
     plugin = PvtPlot(
-        app,
         shared_settings["HM_SETTINGS"],
         ensembles=shared_settings["HM_ENSEMBLES"],
         pvt_relative_file_path="share/results/tables/pvt.csv",
```

Comment:
This whole file must be adjusted to WLF. This is a task that must be performed for all tests and we should implement convenient functions for this. We should consider if we want to postpone this or to hold this PR until the framework is properly implemented.

Reply:
Since not all plugins are rebuild with the new framework, maybe the latter one is a good idea.